src/components/styles/demo/demo.module.css

```diff
@@ -0,0 +1,47 @@
+.demo-section-top {
+  background-color: var(--cornslik);
+  height: 50%;
+  color: var(--beige);
+  position: absolute;
+  right: 0;
+  transform: translateX(50%);
+  padding: 6% 5%;
```

Comment:
Uważałbym z paddiniegm wyrażonym w `%`. Na różnych wielkościach ekranu będziesz miał różne odstępy. Chyba, że jest to zamierzone to niech zostanie